app/views/jobs/show.html.erb

```diff
@@ -10,4 +12,34 @@ Level of Interest: <%= @job.level_of_interest %>
 City: <%= @job.city %>
 <br>
 
+Category: <%= link_to @job.category.title, category_path(@job.category) %>
+<br>
+
 Company: <%= link_to @job.company.name, company_path(@job.company) %>
+<br />
+
+<%= link_to "Edit", edit_company_job_path(@company, @job) %><br />
+<%= link_to "Delete", company_job_path(@company, @job), method: :delete, data: {confirm: "Are you sure you want to delete this job?"} %>
+<br />
+<br />
+
+Comments:<br />
+
+<% if @job.comments.count > 0 %>
```

Comment:
this is a good place for either a helper method or another method in your model. as stands, this is too much logic in your view. your view must do this calculation before executing this code